ddr/include/ddr/ir/Macro.hpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2015, 2017 IBM Corp. and others
+ * Copyright (c) 2015, 2018 IBM Corp. and others
```

Comment:
Please revert this change.

Reply:
> Please revert this change.

From an outside perspective, any reason why? Was this file crafted in 2017 and unchanged in the new year?

Reply:
That's right; the copyright date only needs to be updated if the file was changed (in some other way).